GTBitsOfGood/angels-among-us
# Feed

## Feed

Issue Number(s): #46.

What does this PR change and why?
- Create feed on / route
- Create state for filters (selected filters are currently console logged)
- Generate filters from a object that holds all the filter text and options and enum values
- Create a post card with hardcoded data for now
- Style for desktop and mobile
- Create API route and db action to get all posts sorted by date

### Checklist


### Critical Changes

- None

### Related PRs

- None

### Testing

- Select/deselect a bunch of filters and make sure that it is reflected in the filter state
- Check that things look correct on desktop/mobile
- Add posts to mongodb and use command ```curl -X GET "http://localhost:3000/api/trpc/post.getAllPosts"```, check that posts are ordered from most to least recent
